src/openzeppelin/governance/timelock/Timelock.cairo

```diff
@@ -0,0 +1,316 @@
+# SPDX-License-Identifier: MIT
+# OpenZeppelin Contracts for Cairo v0.1.0 (governance/timelock/Timelock.cairo)
+
+%lang starknet
+
+from starkware.cairo.common.uint256 import Uint256
+from starkware.cairo.common.bool import TRUE, FALSE
+from starkware.cairo.common.cairo_builtins import HashBuiltin
+from starkware.starknet.common.syscalls import get_caller_address, get_contract_address
+
+from openzeppelin.utils.constants import IERC721_RECEIVER_ID
+from openzeppelin.governance.timelock.library import Timelock, TimelockCall
+from openzeppelin.introspection.ERC165 import ERC165_supports_interface, ERC165_register_interface
+
+from openzeppelin.access.accesscontrol import (
+    _grantRole,
+    _setRoleAdmin,
+    AccessControl_hasRole,
+    AccessControl_onlyRole,
+    AccessControl_grantRole,
+    AccessControl_revokeRole,
+    AccessControl_initializer,
+    AccessControl_renounceRole,
+    AccessControl_getRoleAdmin,
+)
+
+
+#
+# Constants
+#
+
+const TIMELOCK_ADMIN_ROLE = 0x1
+const PROPOSER_ROLE = 0x2
+const CANCELLER_ROLE = 0x3
+const EXECUTOR_ROLE = 0x4
+
+
+@constructor
+func constructor{
+    syscall_ptr: felt*, 
+    pedersen_ptr: HashBuiltin*, 
+    range_check_ptr
+}(
+    proposer: felt, 
+    executor: felt, 
+    delay: felt
+):
+    alloc_locals
+
+    ERC165_register_interface(IERC721_RECEIVER_ID)
+
+    AccessControl_initializer()
+    Timelock.initializer(delay)
+
+    _setRoleAdmin(TIMELOCK_ADMIN_ROLE, TIMELOCK_ADMIN_ROLE)
+    _setRoleAdmin(PROPOSER_ROLE, TIMELOCK_ADMIN_ROLE)
+    _setRoleAdmin(CANCELLER_ROLE, TIMELOCK_ADMIN_ROLE)
+    _setRoleAdmin(EXECUTOR_ROLE, TIMELOCK_ADMIN_ROLE)
+
+    let (caller) = get_caller_address()
+    let (this) = get_contract_address()
+
+    _grantRole(TIMELOCK_ADMIN_ROLE, this)
+    _grantRole(TIMELOCK_ADMIN_ROLE, caller)
+
+    _grantRole(PROPOSER_ROLE, proposer)
+    _grantRole(CANCELLER_ROLE, proposer)
+    _grantRole(EXECUTOR_ROLE, executor)
+
+    return ()
+end
+
+@view
+func isOperation{
+    syscall_ptr: felt*, 
+    pedersen_ptr: HashBuiltin*, 
+    range_check_ptr
+}(id: felt) -> (is_operation: felt):
```

Comment:
```suggestion
}(id: felt) -> (isOperation: felt):
```
this should be camelCase. Also, note that the library now uses `operation` over `is_operation`